test/e2e/kubectl.go

```diff
@@ -205,7 +207,7 @@ var _ = framework.KubeDescribe("Kubectl alpha client", func() {
 		})
 
 		It("should create a ScheduledJob", func() {
-			framework.SkipIfMissingResource(f.ClientPool, ScheduledJobGroupVersionResource, f.Namespace.Name)
+			framework.SkipIfMissingResource(f.ClientPool, workload.ScheduledJobGroupVersionResource, f.Namespace.Name)
```

Comment:
Instead of importing the whole package just for these two variables, copy them here.